node-graph/interpreted-executor/Cargo.toml

```diff
@@ -8,6 +8,7 @@ license = "MIT OR Apache-2.0"
 default = []
 serde = ["dep:serde", "graphene-std/serde", "glam/serde"]
 gpu = ["graphene-std/gpu"]
+quantization = ["graphene-std/quantization"]
```

Comment:
How come this is not enabled by default?

Reply:
It pulls in extra dependencies which I didn't want to slow down compile time by default.
+ this is mostly experimental still I just wanted to include this in the main repo to reduce the number of diverging branches